amplfi/train/testing.py

```diff
@@ -27,7 +27,8 @@ def get_sky_projection(ra, dec, dist, nside=32, min_samples_per_pix=15):
     """
     theta = np.pi / 2 - dec
     # mask out non physical samples;
-    mask = (ra > 0) * (ra < 2 * np.pi)
+    # mask = (ra > 0) * (ra < 2 * np.pi)
+    mask = (ra > -np.pi) * (ra < np.pi)
```

Comment:
Let's remove this from version control, and stick to the 0 to 2*pi convention (same as ligo.skymap). If you have a trained model that you are working with, you can apply this patch locally.